src/pages/faq.js

```diff
@@ -19,7 +19,8 @@ function Faq(props) {
   `
```

Comment:
Can't comment the correct line, but a few lines above you define a styled component inside your Faq component.
This is not related to this PR.

Reply:
I had already fixed this issue in my other branch.